test/autorest/generated/lrogroup/lroretrys.go

```diff
@@ -60,25 +46,36 @@ func (client *lroRetrysOperations) BeginDelete202Retry200(ctx context.Context) (
 	if err != nil {
 		return nil, err
 	}
-	pt, err := createPollingTracker("lroRetrysDelete202Retry200Poller", resp, client.delete202Retry200HandleError)
+	result, err := client.delete202Retry200HandleResponse(resp)
 	if err != nil {
 		return nil, err
 	}
-	return &lroRetrysDelete202Retry200Poller{
-		pt:     pt,
-		client: client,
-	}, nil
-}
-
-func (client *lroRetrysOperations) ResumeLroRetrysDelete202Retry200Poller(token string) (LroRetrysDelete202Retry200Poller, error) {
-	pt, err := resumePollingTracker("lroRetrysDelete202Retry200Poller", token, client.delete202Retry200HandleError)
+	pt, err := createPollingTracker("HTTPPoller", resp, client.delete202Retry200HandleError)
 	if err != nil {
 		return nil, err
 	}
-	return &lroRetrysDelete202Retry200Poller{
-		client: client,
-		pt:     pt,
-	}, nil
+	result.GetPoller = func() HTTPPoller {
```

Comment:
I just had a thought.  Does this really need to be a func, couldn't it just be a field of type `HTTPPoller`?  Then we can do this.
```go
	result.GetPoller = &httpPoller{
		pt:       pt,
		pipeline: client.p,
	}
```
Something to discuss in scrum tomorrow.

Reply:
Yes I dont see why not.

Reply:
Done